pinaxcon/templates/dashboard.html

```diff
@@ -43,7 +43,7 @@ <h3 class="panel-title">
                     <a href="{% url "speaker_edit" %}" class="btn btn-xs btn-default">
                         <i class="fa fa-pencil"></i> Edit your speaker profile
                     </a>
-                    <a href="{{ URL_PREFIX }}/proposals/submit/talk" class="btn btn-xs btn-default">
+                    <a href="{{ URL_PREFIX }}/proposals/submit" class="btn btn-xs btn-default">
```

Comment:
Does this cause it to make you select which type of submission on the next page?

Reply:
yep

Reply:
I can switch it to having talk and tutorial buttons if we prefer.

Reply:
Actually, yeah. Let's do separate buttons.